src/index.ts

```diff
@@ -248,6 +246,19 @@ task("starknet-compile", "Compiles Starknet contracts")
         }
     });
 
+async function traverseFiles(traversable: string, fileCriteria: string = "*") {
+    let paths: string[] = [];
+    if (fs.lstatSync(traversable).isDirectory()) {
+        const globPromise = promisify(glob);
```

Comment:
Instead of promisifying this on each `traverseFiles` call, I'd suggest creating a global const function right after the imports at the top of the file.